code/modules/client/preferences_toggles.dm

```diff
@@ -1,10 +1,16 @@
-/client/verb/setup_character()
+/client/verb/game_preferences()
 	set name = "Game Preferences"
 	set category = "Preferences"
 	set desc = "Open Game Preferences Window"
 	prefs.current_tab = 1
 	prefs.ShowChoices(usr)
 
+/client/verb/character_preferences()
+	set name = "Character Preferences"
+	set category = "Preferences"
+	set desc = "Open Character Preferences Window"
+	prefs.ShowChoices(usr)
```

Comment:
don't you have to set current_tab to 0 or does showchoices do this?

Reply:
Pretty sure it's 0 by default.

Reply:
Eh screw it I'll just make it be 0